Fill doctrine/orm bridge documentation (#141)
yokai-php/batch-src: docs/docs/bridges/doctrine-orm.rst

Bridge with ``doctrine/orm``
============================================================

> See `official documentation <https://www.doctrine-project.org/projects/orm.html>`__ on Doctrine's website.

Reviewer: ```suggestion
Refer to the  `official documentation <https://www.doctrine-project.org/projects/orm.html>`__ on Doctrine's website.
```

Read from entities in database
------------------------------------------------------------

> The reader will yield every entity of certain class, one after the other.

Reviewer: ```suggestion
The reader will yield each entity of a specified class, one at a time.
```
